sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -2325,66 +2153,14 @@ def get_transceiver_status(self):
-                trans_status['prefecberhighwarning_flag'] = self.vdm_dict['Pre-FEC BER Average Media Input'][1][7]
-                trans_status['prefecberlowwarning_flag'] = self.vdm_dict['Pre-FEC BER Average Media Input'][1][8]
-                trans_status['postfecberhighalarm_flag'] = self.vdm_dict['Errored Frames Average Media Input'][1][5]
-                trans_status['postfecberlowalarm_flag'] = self.vdm_dict['Errored Frames Average Media Input'][1][6]
-                trans_status['postfecberhighwarning_flag'] = self.vdm_dict['Errored Frames Average Media Input'][1][7]
-                trans_status['postfecberlowwarning_flag'] = self.vdm_dict['Errored Frames Average Media Input'][1][8]
-            except (KeyError, TypeError):
-                pass
         return trans_status
 
     def get_transceiver_status_flags(self):
         """
         Retrieves transceiver status flags of this SFP
 
         Returns:
-            A dict which contains following keys/values :
-        ================================================================================
-        ; Defines Transceiver Status info for a port
-        key                                     = TRANSCEIVER_STATUS_FLAG|ifname        ; Flag information for module on port
-        ; field                                 = value
-        datapath_firmware_fault                 = BOOLEAN           ; datapath (DSP) firmware fault
-        module_firmware_fault                   = BOOLEAN           ; module firmware fault
-        module_state_changed                    = BOOLEAN           ; module state changed
-        txfault{lane_num}                       = BOOLEAN           ; tx fault flag on media lane {lane_num}
-        txlos_hostlane{lane_num}                = BOOLEAN           ; tx loss of signal flag on host lane {lane_num}
-        txcdrlol_hostlane{lane_num}             = BOOLEAN           ; tx clock and data recovery loss of lock flag on host lane {lane_num}
-        tx_eq_fault{lane_num}                   = BOOLEAN           ; tx equalization fault flag on host lane {lane_num}
-        rxlos{lane_num}                         = BOOLEAN           ; rx loss of signal flag on media lane {lane_num}
-        rxcdrlol{lane_num}                      = BOOLEAN           ; rx clock and data recovery loss of lock flag on media lane {lane_num}
-        target_output_power_oor                 = BOOLEAN           ; target output power out of range flag
-        fine_tuning_oor                         = BOOLEAN           ; fine tuning  out of range flag
-        tuning_not_accepted                     = BOOLEAN           ; tuning not accepted flag
-        invalid_channel_num                     = BOOLEAN           ; invalid channel number flag
-        tuning_complete                         = BOOLEAN           ; tuning complete flag
-        ================================================================================
+            Dictionary
```

Comment:
@mihirpat1 add comment in code how this is different from the get_transceiver_status() ?

Reply:
@prgeor Addressed this now